angular/core/components/group_page/group_theme/group_theme.coffee

```diff
@@ -12,7 +12,7 @@ angular.module('loomioApp').directive 'groupTheme', ->
       { 'background-image': "url(#{$scope.group.coverUrl()})", 'z-index': (-1 if $scope.compact) }
 
     $scope.isMember = ->
-      CurrentUser.membershipFor($scope.group)?
+      Session.current().membershipFor($scope.group)?
```

Comment:
At first sight, I find it annoying that this is not `Session.user()` I thought that's what you were going to go with.